### ISSUE TITLE
No internet after standby/hybernate

### ISSUE THREAD

**Issue by AtzeDeVries** [The-Compiler/qutebrowser#1233]
Hi,
 
If my laptop returns from standby or hybernate, current open tabs are niot getting 'internet' anymore. Reloading does not work (getting unable to load page error). Opening the same page in a new (not a other) tab works fine.

```
OS: Archlinux
Hardware: Dell XPS 13

version info
--- ~ » qutebrowser --version
qutebrowser v0.5.0
Git commit: 

CPython: 3.5.1
Qt: 5.5.1, runtime: 5.5.1
PyQt: 5.5.1
```

**Comment by NoSuck** [comment]
This sounds like a repeat of #1054.

**Comment by The-Compiler** [comment]
Indeed. This is a Qt issue which should be fixed when Qt 5.6 comes out.

**Comment by AtzeDeVries** [comment]
ah you are right, i didn't search for suspend resume instead of hybernate. My bad
